org-configurations/default-configuration.yml

```diff
@@ -340,6 +340,11 @@ client:
             en: "password"
           label:
             en: "password"
+        remember_me:
+          type: "checkbox"
+          value: true
+          label:
+            en: "Automatically log me in to the WiFi service at the next visit"
```

Comment:
after testing it I noticed it's too long, can you please shorten this and the other occurrences to something more conservative like: `Remember me on this device`?